app.js

```diff
@@ -0,0 +1,12 @@
+const http = require('http');
+require('dotenv').config()
+const host = process.env.NODE_ENV === "local" ? process.env.LOCAL_HOST : process.env.LIVE_HOST;
+
+const server = http.createServer((req, res) => {
+    if(req.url == '/'){
+        res.write('it is Home Page');  
```

Comment:
send a JSON response. this is an API and will not be going to a page directly. there will be no home page for this.